README.md

```diff
@@ -125,7 +125,7 @@ This mode requires you to write another TOML file. [gtk/Gir.toml](https://github
 
 ```toml
 [options]
-girs_dir = "gir-files"
+girs_directories = ["gir-files"]
```

Comment:
Can we make it so that existing `Gir.toml` that just have a string continue to work? I.e. you'd check if there's a string or an array and use them accordingly? Seems like useless churn to have to update this everywhere.

Reply:
Well, since we're updating the name of the option in any case...

Reply:
Can we maybe continue allowing the old one then? Doesn't seem like a very useful change otherwise

Reply:
I'd rather not so that this parameter remains the reflection of the command-line argument. If you're just bored to make the change in gstreamer-rs, I'll send you a PR. But if it's more than that, we can see what @bilelmoussaoui thinks about it too (and maybe other gir users too).

Reply:
I can change it just with sed, but it seems like potentially confusing for all the outside users of gir. What would the behaviour be when running it with the old name?

Reply:
I can add an error saying that the parameter was renamed?

Reply:
Why not just call it `sources` instead of girs_directories ?

Reply:
`sources` could be about almost anything.

Reply:
Sure, but it's in a Gir.toml file and it's literally the only input you give to gir (as in files) ? but I agree that sources isn't very clear. But I find girs_directories to be extra-verbose.

Reply:
You only need to write it once, you'll survive. :p And better extra-verbose than super confused.

Reply:
I've just removed this entirely because it makes little sense:

- Hard to do proper path "canonicalization" (without getting a full root path);
- No need to find-replace the `../gir-files`, `../../gir-files` and `../../../gir-files` variants;
- Already passing these paths with `-d` in `generator.py`, those are relative to the root directory already.
  Besides, these _override_ the value set by `girs_dirs`!!(!!)